barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -159,17 +159,8 @@ void ProtoGalaxyVerifier_<VerifierInstances>::prepare_for_folding(const std::vec
         receive_accumulator(inst, domain_separator);
     } else {
         // This is the first round of folding and we need to generate some gate challenges.
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/740): implement option 2 to make this more
-        // efficient by avoiding the computation of the perturbator
         receive_and_finalise_instance(inst, domain_separator);
         inst->target_sum = 0;
```

Comment:
could also maybe get rid of this, but its not that important since its one value

Reply:
That’s the issue you are resolving I belive

Reply:
no, sorry, its referring to the target_sum

Reply:
No you cannot do that, the verifier should initialise the first coefficient of the perturbator with the target sum of the accumulator